spec/declaration.dd

```diff
@@ -502,6 +506,33 @@ a = 2;         // sets `S.i` to `2`
 b = 4;         // sets `S.j` to `4`
 -----------
 
+$(H3 $(LNAME2 alias-template-declaration, Alias Template Declarations))
+
+    $(P If the optional $(GLINK2 template, TemplateParameters) is present, then the
+    $(GLINK AliasDeclaration):)
+
+$(GRAMMAR
+    $(D alias) $(GLINK_LEX Identifier) $(GLINK TemplateParameters) $(D =) $(GLINK AliasRHS) $(D ;)
+)
+
+    $(P is a shorthand syntax for a $(GLINK2 template, TemplateDeclaration) of the form:)
+
+$(GRAMMAR
+    $(D template) $(GLINK_LEX Identifier) $(GLINK2 template, TemplateParameters) $(D { alias) $(GLINK_LEX Identifier) $(D =) $(GLINK AliasRHS) $(D })
```

Comment:
Perhaps here?

Reply:
I have no idea.

Reply:
Oh that's definitely a problem. $(D ...) means just code, don't put other things in there.

Reply:
ok